wdl/validate.wdls.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Does the latest version of cromshell do this validation for us (cc: @jonn-smith)?  Is it necessary to have this separate checking script?

Reply:
If you have `womtool` on your path (i.e. if you installed `cromwell` with `homebrew`) then it will do validation, otherwise it doesn't know how.

It's probably still useful to have a stand-alone validation script in the cases where people don't have `cromwell` installed.